content/blog/hello-world.md

```diff
@@ -1,28 +1,45 @@
+With the increasing reliance of scientific research on infrastructural and analytical software, foundational tools in life sciences have outgrown individual labs and institutes in terms of their scale and impact. That has encouraged us to start something more than a code repository for an exciting new project – we are starting a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem.
 
-Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our scope and focus in the very beginning of our journey that we have called {{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}}.
+Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our initial scope and focus in the very beginning of our journey. We are excited to announce **scverse** – a consortium for joint maintenance of core packages in omics data analysis in life sciences.
 
+## What tools are part of scverse?
 
-## What this is about
+We start with the two most important data structures, [AnnData](https://github.com/scverse/anndata) and [MuData](https://github.com/scverse/mudata), together with [Scanpy](https://github.com/scverse/scanpy) for general purpose single-cell analysis, [muon](https://github.com/scverse/muon) for multimodal single-cell analysis, [scvi-tools](https://github.com/scverse/scvi-tools) for deep probabilistic analysis of single-cell data and [scirpy](https://github.com/scverse/scirpy) for T-cell receptor analysis. We are looking forward to expanding with more major libraries and frameworks which fit our vision.
 
-{{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}} is a consortium for joint maintenance of core packages in omics data analysis in life sciences. It has been founded to ensure the long-term maintenance of the core infrastructure in the field.
+## What are we doing?
 
+There are already a few visible outcomes of our joint work including improved 
+[mudata performance](https://mudata.readthedocs.io/en/latest/changelog.html#v0-1-2) and
+[mudata documentation](https://mudata.readthedocs.io/en/latest/). Additionally, 
+[scvi-tools is now using MuData](https://github.com/scverse/scvi-tools/pull/1444) for the representation of multimodal data. 
+[Scanpy has gained new workshop notebooks](https://github.com/scverse/scanpy-tutorials/pull/52), and, moreover, we have started 
+[a new page with tutorials for scverse libraries](https://scverse.org/tutorials/).
```

Comment:
these are all niche. I would think that the largest audience for this post will be people who use our software. Thus, I would start with how the first priority is really building the community, as even what we've done so far is impossible without this. Like I'd start:

1. Our primary goal is to improve how the packages work together etc etc
2. To achieve this we'd like to both build a community of developers, but more importantly users to help us direct the software in the long-term
3. Then I would discuss things we've done in the short term that this has enabled